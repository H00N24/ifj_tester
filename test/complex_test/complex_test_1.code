// Rekurzia

class Main
{
	static int a = 10;

	static void run()
	{
		int v = Druha.rekurz();
		ifj16.print("Vysledok: " + v + "\n");
	}
}

class Druha
{
	static int b = Main.a;
	static int rekurz()
	{
		if (b > 5)
		{
			b = b - 1;
			ifj16.print("b: " + b + "\n");
			return rekurz();
		}

		return 258;
	}
}
