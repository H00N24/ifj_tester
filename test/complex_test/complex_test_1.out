b: 9
b: 8
b: 7
b: 6
b: 5
Vysledok: 258
